webex_skills/cli/skill.py

```diff
@@ -85,29 +85,30 @@ def invoke_skill(
     device_id: str,
     verbose: bool = False,
 ):
-    challenge = os.urandom(32).hex()
     default_params = {
         'time_zone': 'UTC',
         'timestamp': datetime.utcnow().timestamp(),
         'language': 'en',
     }
-    message = {
-        'challenge': challenge,
+
+    default_context = {
+        'userId': user_id,
+        'orgId': org_id,
+        'developerDeviceId': device_id,
+    }
+
+    req = {
+        'challenge': (os.urandom(32).hex()),
```

Comment:
any reason for the extra parens here?

Reply:
Nope, looks like PyCharm added them when I did 'inline variable' and I didn't notice. Removed now